src/types/UiaTracing.h

```diff
@@ -88,9 +88,9 @@ namespace Microsoft::Console::Types
         UiaTracing& operator=(const UiaTracing&) = delete;
         UiaTracing& operator=(UiaTracing&&) = delete;
 
-        static std::wstring _getValue(const ScreenInfoUiaProviderBase& siup) noexcept;
-        static std::wstring _getValue(const UiaTextRangeBase& utr) noexcept;
-        static std::wstring _getValue(const TextPatternRangeEndpoint endpoint) noexcept;
-        static std::wstring _getValue(const TextUnit unit) noexcept;
+        static inline std::wstring _getValue(const ScreenInfoUiaProviderBase& siup) noexcept;

```

Comment:
if they're static _inline_ they don't even need to be in the header.. but also i don't care